packages/ai-assistant/src/app/api/chat/controller.py

```
@@ -0,0 +1,43 @@
+        # 3. Gọi hàm xử lý của ChatService
+        # (Truyền thêm session_id / user_id nếu service của bạn dùng Memory/LangGraph)
+        ai_response = chat_service.process_query(message, session_id)
+
+        # 4. Trả kết quả chuẩn JSON
+        return jsonify({
+            "message": message,
+            "response": ai_response,
+            "session_id": session_id
+        }), 200
+
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)